README.md

```diff
@@ -24,7 +24,19 @@ Options:
     -v, --version       Print version and exit
 ```
 
-To use with slop (as a replacement for `maim -s`):
+## Examples
+
+#### To use with hacksaw: take a screenshot and copy to clipboard
+```sh
+#!/usr/bin/env bash
```

Comment:
Isn't this POSIX-compatible? Why not just `#!/bin/sh`?

Reply:
yeah, it could be that. I wasn't checking if what I wrote was POSIX sh.